packages/alert-dialog/README.md

```diff
@@ -201,3 +247,26 @@ An alert dialog can have a total of 3 buttons if the secondary outline button la
     </sp-button>
 </sp-alert-dialog>
 ```
+
+### Accessibility
+
+The `<sp-alert-dialog>` component follows these accessibility guidelines:
+
+-   Uses semantic heading elements (`<h2>`) for the dialog title
+-   Ensures all buttons have clear, descriptive labels
+-   Supports keyboard navigation:
+    -   <kbd>Tab</kbd>: Move focus between buttons
+    -   <kbd>Space</kbd>/<kbd>Enter</kbd>: Activate the focused button
+    -   <kbd>Esc</kbd>: Close the dialog
+-   Maintains proper focus management when opened and closed
+-   Uses ARIA roles and attributes appropriately:
```

Comment:
Should our examples have `role` defined based on this guidance?

Reply:
@caseyisonit IMHO this should be part of what the component does for the consumer, rather than the onus be on the consumer. Also, I'm interested in having a11y auditors confirm that modal alerts are `alertdialog` because they interrupt user action where `alert` roles do not.